lib/Settings.ts

```diff
@@ -14,13 +14,6 @@ export const getServerSettingValue = async (read: IRead, id: string) => {
 export const getLivechatAgentConfig = async (read: IRead, sessionId: string, type?: string) => {
 
     try {
-        const persistentAgentConfig = await getPersistentAgentConfigToRoom(read, sessionId);
-        if (persistentAgentConfig) {
-            if (type) {
-                return persistentAgentConfig[type];
-            }
-            return persistentAgentConfig;
-        }
 
```

Comment:
@bhardwajaditya This affects connection draining functionality from this PR https://github.com/WideChat/Apps.Dialogflow/pull/120.

The setting should be reflected for new chats/sessions. For the ongoing chats/sessions it should have the same setting when the chat was started

Reply:
@Shailesh351 if I open the widget and then change the settings, it returns the old setting value only in all the chats for that session.

I will test what happens when I open the widget in incognito. If the settings change work fine there , is that an acceptable situation?

Reply:
@Shailesh351 is correct about 'connection draining'.  Any open rooms should continue to use the same config that started their conversation.  Any new rooms should use the new config. 

Unfortunately, the behavior that I have observed is flaky.  It frequently does not take the new settings until you restart the server, but sometimes it works.......  flaky......